src/CommunityToolkit.Maui.Analyzers.UnitTests/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -6,11 +6,11 @@ namespace CommunityToolkit.Maui.Analyzers.UnitTests;
 public static partial class CSharpAnalyzerVerifier<TAnalyzer>
 	where TAnalyzer : DiagnosticAnalyzer, new()
 {
-	public class Test : CSharpAnalyzerTest<TAnalyzer, Microsoft.CodeAnalysis.Testing.DefaultVerifier>
+	class Test : CSharpAnalyzerTest<TAnalyzer, Microsoft.CodeAnalysis.Testing.DefaultVerifier>
 	{
 		public Test(params ReadOnlySpan<Type> assembliesUnderTest)
 		{
-#if NET9_0
+#if NET10_0
 			ReferenceAssemblies = Microsoft.CodeAnalysis.Testing.ReferenceAssemblies.Net.Net90;
```

Comment:
Should we also update net10 here?

Reply:
Yup! We're waiting on Microsoft to release support for testing Analyzers with .NET 10. `Microsoft.CodeAnalysis.Testing.ReferenceAssemblys.Net.Net100` hasn't yet been released. 

For now, the Analyzer tests all fail. 

I assume (hope) Microsoft will push support for this before .NET 10 GAs.